ansible_collections/arista/avd/examples/campus-fabric/README.md

```diff
@@ -0,0 +1,390 @@
+| DC1_SPINES.yml           | Device type for Spines                        |
+| DC1_LEAFS.yml            | Device type for Leafs                         |
+| DC1_NETWORK_SERVICES.yml | VLANs/SVIs                                    |
+| DC1_NETWORK_PORTS.yml    | Port Profiles and Network Port Ranges         |
+
+The tabs below show the Ansible **group_vars** used in this example.
+
+=== "DC1"
+    At the top level (DC1), the following variables are defined in **group_vars/DC1.yml**. These Ansible variables apply to all nodes in the fabric and is a common place to set AAA, users, NTP, and management interface settings. Update local_users and passwords for your environment.
+
+    You can create a sha512_password by creating a username and password on a switch and copy/paste it within double quotes here.
+
+    ``` yaml
+    --8<--
+    examples/campus-fabric/group_vars/DC1.yml
+    --8<--
+    ```
+
+=== "DC1_FABRIC"
+    At the Fabric level (DC1_FABRIC), the following variables are defined in **group_vars/DC1_FABRIC.yml**. The fabric name, design type (l2ls), spine and leaf defaults, ansible authentication, and interface links are defined at this level. Other variables you must supply include: spanning-tree mode and priority along with an mlag IP pool. Typically, and IDF has a unique set of VLANs. You may use the `filter.tags` variable to constrain which vlans are built on an IDF node.
+
+    Variables applied under the node key type (spine/leaf) defaults section are inherited to nodes under each type. These variables may be overwritten under the node itself.
+
+    The spine interface used by a particular leaf is defined from the leaf's perspective with a variable called `uplink_switch_interfaces`. For example, LEAF2A has a unique variable `uplink_switch_interfaces: [Ethernet49/1, Ethernet49/1]` defined. This means that LEAF2A is connected to SPINE1's Ethernet49/1 and SPINE2's Ethernet49/1, respectively.
+
+    ``` yaml
+    --8<--
+    examples/campus-fabric/group_vars/DC1_FABRIC.yml
+    --8<--
+    ```
+
+=== "DC1_SPINES"
+    In an L2LS design, there are two types of spine nodes: `spine` and `l3spine`. In AVD. the node type defines the functionality and the EOS CLI configuration to be generated. For an L2LS design, we will use node type: spine. Later, we will add routing to the Spines by changing the node type to l3spine.
+
+    ``` yaml
+    --8<--
+    examples/campus-fabric/group_vars/DC1_SPINES.yml
+    --8<--
+    ```
+
+=== "DC1_LEAFS"
+    In an L2LS design, we have one type of leaf node: `leaf`.
+
+    ``` yaml
+    --8<--
+    examples/campus-fabric/group_vars/DC1_LEAFS.yml
+    --8<--
+    ```
+
+=== "DC1_NETWORK_SERVICES"
+    You add VLANs and SVIs to the Fabric by updating the **group_vars/DC1_NETWORK_SERVICES.yml**. Each VLAN will be given a name and a list of tags. The tags filter the VLAN to specific Leaf Pairs. These variables are applied to spine and leaf nodes since they are a part of this group.
```

Comment:
```suggestion
    You add VLANs and SVIs to the Fabric by updating the **group_vars/DC1_NETWORK_SERVICES.yml** file. Each VLAN will be given a name and a list of tags. The tags filter the VLAN to specific Leaf Pairs. These variables are applied to spine and leaf nodes since they are a part of this group.
```

Reply:
done